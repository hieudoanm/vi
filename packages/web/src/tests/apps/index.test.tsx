import { UserProvider } from '@auth0/nextjs-auth0';
import { render } from '@testing-library/react';
import AppsPage from '../../pages/apps';

describe('AppsTemplate', () => {
  test('render default', () => {
    const wrapper = render(
      <UserProvider>
        <AppsPage />
      </UserProvider>
    );
    expect(wrapper.container).toMatchSnapshot();
  });
});
