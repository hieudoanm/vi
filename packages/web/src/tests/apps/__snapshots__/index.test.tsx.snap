// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppsTemplate render default 1`] = `
<div>
  Loading
</div>
`;
